Aztec/Classes/TextKit/TextView.swift

```diff
@@ -223,6 +223,12 @@ open class TextView: UITextView {
     // MARK: - Properties: Text Lists
 
     var maximumListIndentationLevels = 7
+    
+    // MARK: - Properties: Blockquotes
+    
+    /// The max levels of quote indentation allowed
+    /// Set to 9 for iPhone sized screen, 24 for other devices
+    var maximumBlockquoteIndentationLevels = UIDevice.current.userInterfaceIdiom == .phone ? 9 : 24
```

Comment:
I think it's better to have a single value here and let users of the library change it according to their needs. 

Also on the iPad we have to support the multitask modes where the screen can have the same width of a phone and 24 levels don't make sense then.

Reply:
hmm, true I wasn't sure how to do the compact width check here. Also if user can change this and it happens to be less than whats in the `blockquoteBorderColors` array it will crash. I think I will just make this value be the count of `blockquoteBorderColors` +9, that should cover any reasonable use case, and on compact widths with high indent, it will just look a little funny, but thats unlikely to happen anyway

Reply:
ok, that makes sense. I will just make it public var then.

Yeah, I wasn't sure how to get compactWidth in that instance didnt spend too much time on it